manager/controlapi/service.go

```diff
@@ -433,7 +433,7 @@ func (s *Server) validateNetworks(networks []*api.NetworkAttachmentConfig) error
 		if network == nil {
 			continue
 		}
-		if network.Spec.Internal {
+		if network.Spec.Ingress {
```

Comment:
Why does this check `Spec.Ingress` directly instead of using `IsIngressNetwork`?